src/test/java/hw10/BoardApiTest.java

```diff
@@ -0,0 +1,76 @@
+package hw10;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.startsWith;
+
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.http.ContentType;
+import org.example.entities.BoardEntity;
+
+public class BoardApiTest extends AbstractApiTest {
+
+    public String endpoint = "https://api.trello.com/1/boards";
```

Comment:
https://api.trello.com/1/ should be added to request spec as base uri, "/boards" added as endpoint. Also you can add /boards as path param to request spec